docs/cas-server-documentation/protocol/CAS-Protocol-Specification.md

```diff
@@ -1525,76 +1525,116 @@ to the CAS client.
 
 ```xml
 <?xml version="1.0" encoding="UTF-8"?>
-<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:cas="http://www.yale.edu/tp/cas" targetNamespace="http://www.yale.edu/tp/cas" elementFormDefault="qualified" attributeFormDefault="unqualified">
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:cas="http://www.yale.edu/tp/cas"
+           targetNamespace="http://www.yale.edu/tp/cas"
+           elementFormDefault="qualified">
+  <xs:annotation>
+    <xs:documentation>
+      The following is the schema for the Central Authentication Service (CAS) version 3.0 protocol response.<br />
+      This covers the responses for the following endpoints: /serviceValidate, /proxyValidate, /p3/serviceValidate, /p3/proxyValidate, /proxy.<br />
+      This specification is subject to change.<br />
+
+      Schema version: 3.0.1<br />
```

Comment:
Should we stay in sync with the protocol spec version (3.0.3 in this case) or increment the schema, only?

Reply:
I recommend we take out the schema version and sync with the protocol version that way.

Reply:
I wouldn't do that. We need the schema as an xsd in the future in the project, if we marshal the XML generation for the serviceValidateResponse instead of hand-crafting them as today using the views. So we should version the schema. Maybe we can externalize the schema into a xsd and include in the MD file (if possible)?

Reply:
Is there a reason CAS protocol X would want to have a different protocol XSD schema version Y implemented by CAS version Z? (You see where I am going with this?)

Reply:
to further elaborate: I am not saying the schema should not be versioned; I am saying it should be, and its version should simply match and be equal to the CAS protocol version. Because it's unlikely (and possibly very confusing) to have a schema version matching one protocol version implemented in one CAS version. In practice, we already have a difficult-enough time explaining to folks "no this is a CAS protocol version; it has nothing to do with the CAS server version".